qutipy/Clifford/__init__.py

```diff
@@ -1,4 +1,250 @@
-from qutipy.Clifford.Clifford_group_generators import Clifford_group_generators
-from qutipy.Clifford.generate_Clifford_group import generate_Clifford_group
-from qutipy.Clifford.generate_state_2design import generate_state_2design
-from qutipy.Clifford.Clifford_twirl_channel_one_qubit import Clifford_twirl_channel_one_qubit
\ No newline at end of file
+#               This file is part of the QuTIpy package.
+#                https://github.com/sumeetkhatri/QuTIpy
+#
+#                   Copyright (c) 2022 Sumeet Khatri.
+#                       --.- ..- - .. .--. -.--
+#
+#
+# SPDX-License-Identifier: AGPL-3.0
+#
+#  This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, version 3.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+
+import numpy as np
+
+from qutipy.channels import apply_channel
+from qutipy.gates import CNOT_ij, H_i, Rx_i, Ry_i, Rz_i, S_i
+from qutipy.general_functions import (
+    dag,
+    eye,
+    ket,
+    trace_distance_pure_states,
+    unitary_distance,
+)
+
+
+def Clifford_group_generators(n):
+    """
+    Outputs the generators of the n-qubit Clifford group.
+    """
+
+    G = []
+
+    if n == 1:
+        G = [H_i(1, 1), S_i(1, 1)]
+    else:
+        for i in range(1, n + 1):
+            G.append(H_i(i, n))
+            G.append(S_i(i, n))
+            for j in range(1, n + 1):
+                if i < j:
+                    G.append(CNOT_ij(i, j, n))
+                else:
+                    continue
+
+    return G
+
+
+def Clifford_twirl_channel_one_qubit(K, rho, sys=1, dim=[2]):
+    """
+    Twirls the given channel with Kraus operators in K by the one-qubit
+    Clifford group on the given subsystem (specified by sys).
+    """
+
+    n = int(np.log2(np.sum([d for d in dim])))
+
```

Comment:
All multiplications in this function need to changed from '*' -> '@'. Make sure Rx_i, Ry_i, Rz_i are returning np.array.

Reply:
Refer to https://github.com/arnavdas88/QuTIpy/blob/master/qutipy/clifford.py#L58